## 06_extend_JIS_example.diff by  <danai@debian.org>
##
## Add extra goodies in the JIS example file.

--- cjk-20060714/examples/JIS.tex	2007-02-05 01:40:40.878024813 +0100
+++ new/examples/JIS.tex	2007-02-05 02:28:49.951124178 +0100
@@ -5,24 +5,40 @@
 %
 % Version 4.8.3 (07-May-2012)
 
-\documentclass[12pt]{article}
+\documentclass[12pt]{scrartcl}
 
 \usepackage{CJK}
 \usepackage[CJK, overlap]{ruby}
 
 \renewcommand{\rubysep}{-0.2ex}
 
-
 \begin{document}
 
 \begin{CJK*}[dnp]{JIS}{min}
 \CJKtilde
+% CJKcaption works only with KOMA-script classes such as "scrartcl".
+\CJKcaption{JIS}
+\title{A Japanese example file for CJK in JIS}
+\author{Danai SAE-HAN（韓達耐）}
+\maketitle
+\tableofcontents
 
-\noindent この~FAQ~リストは、よくある質問とその答を集め、役に立つよう
+\newcommand{\testpar}{この~FAQ~リストは、よくある質問とその答を集め、役に立つよう
 にしたものです。この~FAQ~リストの構造は、以前のものと比べて大幅に変更
 されています。\textbf{\ruby{新}{あたら}しい構造に関しては、「この~FAQ~
 の読み方とその構造」の項目を\ruby{参}{さん}\ruby{照}{しょう}して下さ
-い。}
+い。}}
+
+\section{Wadalab Mincho 明朝}
+\testpar
+
+\section{Wadalab Gothic ゴシック}
+\CJKfamily{goth}
+\testpar
+
+\section{Wadalab Maru 丸}
+\CJKfamily{maru}
+\testpar
 
 \end{CJK*}
 
